<!-- -*- mode: XML -*- -->

<launch>
    <arg name="joy_dev" default="/dev/input/js0" />

    <node name="joy" pkg="joy" type="joy_node" output="screen">
      <param name="dev" value="$(arg joy_dev)" />
    </node>

    <node name="joy_bridge" pkg="delta_manip" type="joy_bridge.py"
    output = "screen"/>
</launch>
